src/mode/modeNormal.ts

```diff
@@ -23,8 +23,10 @@ export class NormalMode extends Mode {
         "gg" : async (c) => { return c.firstLineNonBlankChar().move(); },
         "G" : async (c) => { return c.lastLineNonBlankChar().move(); },
         "w" : async (c) => { return c.wordRight().move(); },
+        "W" : async (c) => { return c.WORDRight().move(); },

```

Comment:
Not a big fan of differentiating functions using casing. I understand that vim differentiates using word vs. WORD, but I'm wondering if there are alternatives?